lib/puppet/util/plist.rb

```diff
@@ -60,7 +61,7 @@ def parse_plist(plist_data, file_path = '')
 
       begin
         plist_obj = new_cfpropertylist(:data => plist_data)
-      rescue CFFormatError => e
+      rescue CFFormatError, NoMethodError => e
```

Comment:
Could you add a comment to the effect of "The CFPropertyList library will raise NoMethodError for invalid data"? It's not entirely clear what's happening here, since we're just wrapping an external library at this point